### DISCUSSION TITLE
i did something like this for site elements.

### DISCUSSION THREAD

**Discussion by fagann** [gazzabwoy/projectEprom#20] [General]
[Proposed Site Elements.docx](https://github.com/gazzabwoy/projectEprom/files/7580184/Proposed.Site.Elements.docx)

**Comment by gazzabwoy** [comment]
I don't think your getting this, unless you want to add and about.html page to navbar and site. Just list the elements that are in the image folder.
form = .html isn't an element that we created, it exists in css. and html5 is the code we write.
text =  home.jpg (jpg is a image extension if its text it should be .txt)
image = home.jpg... we don't have a home.jpg... 4 weeks for this Fagan.. come on, you can do better. 

![image](https://user-images.githubusercontent.com/91916706/142849167-0c086478-5ee0-4bc3-9923-f503ff17b9d4.png)
